dotty-community-build/src/test/scala/dotty/communitybuild/CommunityProjectTest.scala

```diff
@@ -15,30 +15,60 @@ case class CommunityProject(
     Paths.get(sys.props("user.dir")).getParent.resolve(name)
 }
 
+object DottyVersion {
+  // NOTE: the "latest" version in the maven-metadata.xxml uses a release from
+  // January because it has no PATCH version (0.1-201701XXX) while the actual latest
+  // releases have a PATCH version (0.1.1-20170NXXX). Until 0.2 is released,
+  // we hardcode this regex to match a 0.1 release with a PATCH version.
+  private val Version = """      <version>(0.1\..*)</version>""".r
+  lazy val latest = sys.env.getOrElse(
+    "DOTTY_VERSION",
+    scala.io.Source
+      .fromURL(
+        "http://repo1.maven.org/maven2/ch/epfl/lamp/dotty_2.11/maven-metadata.xml")
+      .getLines()
+      .collect {
+        case Version(version) => version
+      }
+      .toSeq
+      .lastOption
+      .getOrElse {
+        throw new IllegalArgumentException(
+          "Unable to automatically find latest build version")
+      }
```

Comment:
scala.io.Source should supposedly not be used, but I find it useful in simple situations like this where I just want to do a GET request and get the http response body back as a string.